public/bootstrap-datepicker/css/bootstrap-datepicker3.css

```diff
@@ -0,0 +1,791 @@
+/*!
```

Comment:
you can google a css to sass converter. i'm googling to see if the CSS and JS files are available through CDN so that we dont have to actually include them in our project.